packages/wdio-shared-store-service/tests/server.test.ts

```diff
@@ -15,12 +15,19 @@ describe('WdioSharedStoreService exports', () => {
         port: number;
         app: PolkaInstance;
     } | undefined
+    let setResourcePoolUrl: string
+    let getValueFromPoolUrl: string
+    let addValueToPoolUrl: string
 
     beforeAll(async () => {
+
         result = await startServer()
         const baseUrl = `http://localhost:${result.port}`
         setUrl = `${baseUrl}/set`
         getUrl = `${baseUrl}/get`
+        setResourcePoolUrl = `${baseUrl}/setResourcePool`
+        getValueFromPoolUrl = `${baseUrl}/getValueFromPool`
+        addValueToPoolUrl = `${baseUrl}/addValueToPool`
```

Comment:
Actually can we change the urls for the internal API to:
```suggestion
        setResourcePoolUrl = `${baseUrl}/pool/set`
        getValueFromPoolUrl = `${baseUrl}/pool/get`
        addValueToPoolUrl = `${baseUrl}/pool/add`
```

Reply:
done, i've only updated the internal api, don't know if you want to also change function names of the JS api